### DISCUSSION TITLE
Codespace won't (re) start - looping between "setting up" to "stopping"

### DISCUSSION THREAD

**Discussion by jeroenminderman** [community/community#136662] [Codespaces]
### Select Topic Area

General

### Body

Hi,
I have an issue with a Codespace; following an attempted rebuild of the container, it now appears to be stuck. On trying to reconnect, it seems to briefly go to "setting up" and then gets stuck on "stopping Codespace".
Please help, am I able to simple re-do the rebuild somehow, manually? I've some uncommitted changes inside it that I'd really like to recover.
Thank you!

**Comment by PremPawar** [comment]
exact same issue i am also facing. how you resolved this issue? pls let me know if any solution is there. Thank you in advance.
